borgmatic/execute.py

```diff
@@ -221,20 +220,29 @@ def execute_command_and_capture_output(
     stdout. If shell is True, execute the command within a shell. If an extra environment dict is
     given, then use it to augment the current environment, and pass the result into the command. If
     a working directory is given, use that as the present working directory when running the command.
+    If raise on exit code one is False, then treat exit code 1 as a warning instead of an error.
 
-    Raise subprocesses.CalledProcessError if an error occurs while running the command.
+    Raise subprocesses.CalledProcessError if an error occurs while running the command, or if the
+    command exits with a non-zero exit code and raise on exit code one is True.
```

Comment:
It looks like this comment is also out of date. I think you can almost just leave it as it was: `Raise subprocesses.CalledProcessError if an error occurs while running the command.` That's because an error is non-zero for all commands except for Borg, for which warnings (exit code 1) are treated as such and errors (not exit code 1 or 0) are treated as errors. So in all cases, you're raising for semantic errors.